pkgs/tools/games/gog-downloader/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, makeWrapper, lib, php }:
```

Comment:
Is that a necessary change? If it's a common format in the repo, I'll be happy to change, even though I myself consider this syntax ugly.

I took my inspiration from [phpstan](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/php-packages/phpstan/default.nix).

Reply:
It's not necessary, but since we are reviewing, we know how things should be done in `nixpkgs`. Not everything is perfect, but we try our best to fix things piece by piece and avoid adding entropy in new PRs.